x-pack/test/api_integration/apis/monitoring/elasticsearch/node_detail.js

```diff
@@ -32,7 +32,7 @@ export default function({ getService }) {
     it('should summarize node with metrics', async () => {
       const { body } = await supertest
         .post(
-          '/api/monitoring/v1/clusters/YCxj-RAgSZCP6GuOQ8M1EQ/elasticsearch/nodes/jxcP6ue7eRCieNNitFTT0EA'
+          '/api/monitoring/v1/clusters/YCxj-RAgSZCP6GuOQ8M1EQ/elasticsearch/nodes/jUT5KdxfRbORSCWkb5zjmA'
```

Comment:
I have no idea why this was changed. The original node id doesn't actually exist in the archived data! See https://github.com/elastic/kibana/pull/23715